packages/gatsby/src/redux/reducers/schema-customization.ts

```diff
@@ -1,6 +1,8 @@
-const initialState = () => {
+import { IGatsbyState, ActionsUnion } from "../types"
+
+const initialState = (): IGatsbyState["schemaCustomization"] => {
   return {
-    composer: null,
+    composer: null as any,
```

Comment:
I'm not aware of a better way to handle this without changing quite a few things around the codebase. Maybe a sensible default could be introduced?